pkg/agent/dispatcher.go

```diff
@@ -165,16 +166,29 @@ func NewDispatcher(
 		var firstSequentialFailure *time.Time
 		continuedFailureAbortTimeout := time.Second * time.Duration(runner.global.config.Monitor.MaxHealthCheckSequentialFailuresSeconds)
 
+		// if we don't have any errors, we will log only every 10th successful health check
+		const maxSuccessiveSkips = 10
+		var successesSkipped int
```

Comment:
WDYT about not resetting this when we log, and instead just counting the number of sequential successes/failures and including the opposite in the logs?

(i.e., on first success, log the number of failures; and vice versa)

Reply:
I didn't get the idea, do think just counting totals would work?

I.e. to maintain `totalSuccessCnt` and `totalFailCnt` and always add them to `logFields`?

Then print successful healthchecks only if `totalSuccessCnt % 10 == 0`?

Reply:
Roughly, yeah- with the additions that

1. `totalSuccessCnt` is reset to 0 on failure
2. `totalFailCnt` is reset to 0 on success
3. Failure-then-success logs `totalFailCnt`
4. Success-then-failure logs `totalSuccessCnt`

Reply:
Ok, made the change.

> Failure-then-success logs totalFailCnt
> Success-then-failure logs totalSuccessCnt

IMO it's not necessary to do this. I pushed a commit now which just counts oks/fails in a row.